components/sections/About.tsx

```diff
@@ -110,8 +113,8 @@ export default function About(): React.ReactElement {
         </Flex>
       </FlexAbout>
       <FlexTagLines
-        flexDirection={["column", "column", "column", "row"]}
-        justifyContent={["center", "center", "center", "space-evenly"]}
+        direction={["column", "column", "column", "row"]}
+        justify={["center", "center", "center", "space-evenly"]}
```

Comment:
Nice Chakra Flex shorthand! maybe we should eventually set up a style convention and refactor everything else to be consistent

Reply:
Yes!